recipe/meta.yaml

```diff
@@ -1,34 +1,47 @@
 # Note: This recipe currently only works on OS X and Linux.
 
 {% set version = "29.3" %}
+{% set gcc_version = "14.2.0" %}
```

Comment:
My concern here is how will/should this get updated? The bot will update the emacs version, but not this. Should we update this whenever emacs is updated? Is there a specific version of gcc that the emacs source recommends pinning against that we can follow?

Reply:
Or does this need to match the version of gcc that emacs was built with? In that case, it should use the conda-forge pinning stuff. It's a little confusing because the package is built with gcc but also separately ships libgccjit.

Reply:
This is independent of the gcc used to build emacs, as long the gcc is able to build gccjit. We could use the same gcc version as installed by conda-build, but this would require downloading the gcc sources manually in build.sh, because bot will not update the source tarball hash anyway, and we would lose conda build's source caching.

Reply:
For a prospective osx build, this is a moot point anyway, as the "main" compiler will be clang, and the JIT compiler needs to be gcc.